website/index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0, maximum-scale=1"
+    />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="./style.css" />
+    <link rel="stylesheet" href="../src/bottomsheet.css" />
+  </head>
+
+  <body>
+    <div class="carousel">
+      <div class="">
+        <h4 class="series">Comedy Series</h4>
+        <ul class="scroll-snap-slider"></ul>
+      </div>
+      <!-- <a href="./maps.html">link</a> -->
+    </div>
+    <script src="https://cdn.jsdelivr.net/npm/animejs@3.2.1/lib/anime.min.js"></script>
+    <!-- <script src="./lib/bottomsheet.umd.js"></script> -->
```

Comment:
We don't need to use the script for anime js for the development page. Since they will be imported in the bottomsheet file in the npm version of the page.

They are only need if we are using the script url version of the bottomsheet file